scripts/run_design.py

```diff
@@ -23,7 +23,11 @@ def run_design(design_path, flow, error_flow, flow_args):
     design = Design(design_path)
 
     # Create temp folder
-    build_dir = Path.cwd() / "build" / flow / (design.path.relative_to(paths.DESIGNS_PATH))
+    print(Path.cwd())
+    print(flow)
+    print(design.path)
+    print(paths.DESIGNS_PATH)
+    build_dir = Path.cwd() / "build" / flow / design.path.parent #change made here
```

Comment:
I don't think this is the right way to fix it.  I think we want to keep the existing `relative_to` code.  But in the case that the design_path is not a subdirectory of DESIGNS_PATH, then we should handle it differently, and we could use `Path.cwd() / "build" / flow / "<external>" / design_path.name`